frontend/model/contracts/identity.js

```diff
@@ -18,10 +22,11 @@ import {
 } from './shared/validators.js'
 
 const attributesType = objectMaybeOf({
-  username: string,
+  username: stringMax(IDENTITY_USERNAME_MAX_CHARS, 'username'),
```

Comment:
Just below this, email should use `IDENTITY_EMAIL_MAX_CHARS` that's equal to `320`